src/main/java/solutions/fairdata/fdp/index/service/EventService.java

```diff
@@ -199,7 +199,7 @@ private void resumeUnfinishedEvents() {
                 } else if (event.getType() == EventType.WebhookTrigger) {
                     webhookService.processWebhookTrigger(event);
                 } else {
-                    logger.warn("Unknown event type " + event.getUuid());
+                    logger.warn("Unknown event type " + event.getUuid() + " (" + event.getType() + ")");
```

Comment:
slf4j loggers can use the `{}` placeholder syntax (which would prevent unnecessary string concatenation):

```suggestion
                    logger.warn("Unknown event type {} ({}) ", event.getUuid(), event.getType());
```

Reply:
Nice catch, fixed